lxd/device/nic_bridged.go

```diff
@@ -1032,15 +1035,15 @@ func (d *nicBridged) setupNativeBridgePortVLANs(hostName string) error {
 		// If the default is different to the specified untagged VLAN or if tagged VLAN is set to "none"
 		// then remove the default untagged membership.
 		if defaultPVID != d.config["vlan"] || d.config["vlan"] == "none" {
-			_, err = shared.RunCommand("bridge", "vlan", "del", "dev", hostName, "vid", defaultPVID)
+			err = link.BridgeVlanDelete(defaultPVID, false, false, false, false)
```

Comment:
@presztak can you follow the convention given in the parent function's name "setupNativeBridgePortVLANs" of upper casing instances of "vlan" as it is an acronym. Thanks